### PR TITLE
Fix mypy pipenv github action

### PR DESCRIPTION
**Pull request by achiar99**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/mypy.yml [hunk 1]
```diff
@@ -12,13 +12,19 @@ jobs:
...
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python
+    - name: Set up Python 3.7
       uses: actions/setup-python@v2
       with:
-        python-version: '3.7'
-    - name: Install Dependencies
+        python-version: 3.7
+    - uses: imranismail/setup-kustomize@v1
```
Comment (gruebel) [inline .github/workflows/mypy.yml:19]: any reason, why we need to setup kustomize?

File: .github/workflows/mypy.yml [hunk 2]
```diff
@@ -12,13 +12,19 @@ jobs:
...
+    - name: Install pipenv
       run: |
-        python -m pip install --upgrade pip
-        pip install mypy
-    - name: Mypy
-      run: mypy
+        python -m pip install --no-cache-dir --upgrade pipenv
+    - name: Install dependencies
+      run: |
+        pipenv --python 3.7
```
[the remark this answers is not in the document]
Comment (achiar99) [reply .github/workflows/mypy.yml:24]: when i remove it the action fail

[the remark this answers is not in the document]
Comment (gruebel) [reply .github/workflows/mypy.yml:24]: ok, then leave it, I actually thought it is not needed 🤔

**Review by gruebel — APPROVED** [review]
🚀

**Pull request merged**